### ISSUE TITLE
Not able to install cupy-cuda111 using poetry

### ISSUE THREAD

**Issue by mr-bjerre** [cupy/cupy#5664]
I use [`poetry`](https://python-poetry.org/) which is a popular python tool for dependency management.

Unfortunately I can't install `cupy-cuda111` using poetry, i.e. `poetry add cupy-cuda111` yields the following error from poetry

```
  RuntimeError

  Unable to find installation candidates for cupy-cuda111 (9.0.0)
```

Version `cupy-cuda101` installs just fine for instance.

#### Workaround

```
poetry add https://github.com/cupy/cupy/releases/download/v9.3.0/cupy_cuda111-9.3.0-cp39-cp39-manylinux1_x86_64.whl
```

**Comment by kmaehashi** [comment]
Hi @mr-bjerre, thanks for the report!
As you may already have noticed, some wheels are not yet uploaded to PyPI due to the disk size limit issue (#4971). We contacted again with the PyPI team to resolve the issue, and as an immediate workaround, we will be removing outdated pre-release wheels from PyPI (#5667) so that we can get recent stable wheels available on PyPI. Sorry for the inconvenience and thank you for the understanding.

**Comment by kmaehashi** [comment]
PyPI disk limit issue has been resolved and v9.3.0 wheels are now on PyPI.

**Comment by mr-bjerre** [comment]
Hi thanks for your quick and insightful response!
